### ISSUE TITLE
players should be able to re-connect after disconnecting

### ISSUE THREAD

**Issue by andy5995** [ZetaGlest/zetaglest-source#5]
If a player lags out, he should be able to re-connect to the game in-progress

**Comment by andy5995** [comment]
@Jammyjamjamman told me the code to do this had been added but was disabled. If it works at all, I suggest enabling it so the bugs can be worked on.

If it's enabled, we'll have to figure out how to test locally if possible, and then document the process.

**Comment by andy5995** [comment]
This enables some code that was put in place by the MG team

https://github.com/ZetaGlest/zetaglest-source/commit/4e9ed100dca8122843ebe533b74143951bb40a92

I tested it. It allows for a player to rejoin after lagging out.

I tested it first by joining a running game

* start client
* host game
* have network slots available when the game starts
* start another client on the same computer
* join LAN game

I then tested rejoining after a lag

* while connected with the 2nd client, I cut my wifi off for a second
* was then dropped by the running host
* was able to rejoin the game

I don't know how stable the code is.. It hasn't been tested for any length of time yet, or under real-life conditions.

Oh, this is one problem.. one can only enable the "in-game" join when setting up a network game from the "custom game" menu, or from the "host game" button in the lobby.
